mackerel-plugin-aws-ec2-ebs/lib/aws-ec2-ebs.go

```diff
@@ -42,62 +46,74 @@ var io1Graphs = append([]string{
 type cloudWatchSetting struct {
 	MetricName string
 	Statistics string
-	CalcFunc   func(float64, float64) float64
+	CalcFunc   func(float64) float64
```

Comment:
I'm curious is there valuable reason to specify other numbers to the period? If nothing, I think it is sufficient to refers `aggregationPeriod` directly in each `CalcFunc`.

Reply:
It's OK!
I feel like it's a bit tightly-coupled and hard to understand (you can't tell why `CalcFunc` divides value by `aggregationPeriod` unless you read the whole program), but it will be no problem because the program itself is fairy small.